sourcemash/api/feeds.py

```diff
@@ -18,14 +20,75 @@
 
 class FeedListAPI(Resource):
```

Comment:
Unsubscribe is currently a delete in FeedAPI. We'd put admin delete in FeedListAPI, and perhaps have the admin delete button send the name of the feed in a form?